code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -61,6 +63,12 @@ GLOBAL_LIST_EMPTY(splatter_cache)
 		deltimer(dry_timer)
 	return ..()
 
+/obj/effect/decal/cleanable/blood/proc/BloodySlip(datum/source, mob/living/carbon/human/H)
+	if(!istype(H))
+		return
+	var/mob/living/carbon/human/slipper = H
+	slipper.slip("the blood", 3, 2, tilesSlipped = 0, walkSafely = 1)
```

Comment:
walkSafely is a boolean value. So `TRUE` here

Reply:
I could make a slippery component, and be all modern and stuff, but screw that! We need more oldschool code, more copy and paste, more 1s and 0s, down with TRUE and FALSE! They are fake variables to hide their true meaning anyway.